src/internal.c

```diff
@@ -7924,14 +7916,36 @@ static INLINE int DtlsUpdateWindow(WOLFSSL* ssl)
     }
 
     if (curLT) {
-        *window |= ((DtlsSeq)1 << (diff - 1));
+        word32 idx = diff / DTLS_WORD_BITS;
+        word32 newDiff = diff % DTLS_WORD_BITS;
+
+        window[idx] |= (1 << (newDiff - 1));
```

Comment:
Should there be a check here that idx (diff / DTLS_WORD_BITS) is less than WOLFSSL_DTLS_WINDOW_WORDS?